GameLauncher/App/MainScreen.cs

```diff
@@ -2183,7 +2183,7 @@ private void settingsGameFiles_Click(object sender, EventArgs e)
                         DownloadModNetFilesRightNow(path);
                     }
                 };
-                client2.Download(url.ToString(), path);
+                client2.DownloadFileAsync(url, path + "/" + FileName);
```

Comment:
Hello, the fix is here, the rest of the code is just a revert to WebClientWithTimeout.